src/version.h

```diff
@@ -9,13 +9,13 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 70015;
+static const int PROTOCOL_VERSION = 70006;
```

Comment:
There is no reason to change ```PROTOCOL_VERSION```.

Reply:
z1.1.0 use this.

```static const int PROTOCOL_VERSION = 70006```

z1.2.1 also use this.

```static const int PROTOCOL_VERSION = 70006```


In order to maintain compatibility, it is better to change the value.
Please tell me your opinion.

Reply:
It is a proposal.

https://github.com/bitzeny/bitzeny/blob/z1.0.x/src/version.h

```static const int PROTOCOL_VERSION = 70004```

https://github.com/bitzeny/bitzeny/blob/z1.1.x/src/version.h

```static const int PROTOCOL_VERSION = 70006```

So,how about [70008] for BitZeny 2.0.0 ?

Reply:
[Discussion result]

PROTOCOL_VERSION = 70015 is better for version control.
I reset the setting to the value before change.